doxy

```diff
@@ -3,122 +3,52 @@
 here=`pwd`
 # echo ${here}
 
-# doxyfile="dox/DoxyHtml"
-# if [ -f ${doxyfile} ]
-# 	then
-# 		doxygen ${doxyfile}
-# fi
-
-# doxyfile="dox/DoxyMan"
-# if [ -f ${doxyfile} ]
-# 	then
-# 		doxygen ${doxyfile}
-# fi
-
+doxyfiles=("DoxyLangman" "DoxyRefman"            "DoxyUserman"      "DoxyStlman")
```

Comment:
I know what you're doing here, but it kills me to see multiple arrays.  Consider....  Making one array of "location;doxyfile;name" or something, then splitting the string apart in the loop.....

Reply:
Given, that's an area of shell scrip I've never messed with, could we relegate it to an issue for now?

Reply:
Okay, I have a better idea.  Why don't you make a shell script function that takes the relevant arguments, then instead of making arrays, you call the function sequentially with the right arguments?  That is probably easier.